go/pkg/p2e/seasons.go

```diff
@@ -170,14 +170,30 @@ var THE_RIOT_SEASONS = []Season{
 		StartsAt:   "2023-08-09T00:00:00",
 		EndsAt:     "2023-09-08T00:00:00",
 	},
-  // Pre-Season 7
-  {
-    ID:        "Pre-Season 7",
-    GameID:    THE_RIOT_GAME_ID,
-    BossName:  `== $*=|'¤?")à-£_%`,
-    BossImage: "https://bafybeiacz7z2kqoskbtixovgzzdwiuhpfc7z4tac37qpxhjxyyljbp7v6i.ipfs.nftstorage.link/",
-    StartsAt:  "2023-09-08T00:00:00",
-    EndsAt:    "2023-12-10T00:00:00",
-    IsPre:     true,
-  },
+	// Pre-Season 7
+	{
+		ID:        "Pre-Season 7",
+		GameID:    THE_RIOT_GAME_ID,
+		BossName:  `== $*=|'¤?")à-£_%`,
+		BossImage: "https://bafybeiacz7z2kqoskbtixovgzzdwiuhpfc7z4tac37qpxhjxyyljbp7v6i.ipfs.nftstorage.link/",
+		StartsAt:  "2023-09-08T00:00:00",
+		EndsAt:    "2023-12-10T00:00:00",
+		IsPre:     true,
+	},
+}
+
+var THE_RIOT_ETHEREUM_SEASONS = []Season{
+	// Season 1
+	{
+		ID:         "Season 1 - ETH",
+		GameID:     THE_RIOT_GAME_ID,
+		Denom:      "eth",
+		Decimals:   18,
+		TotalPrize: 1,
+		BossName:   "Philipp Rustov",
+		BossImage:  "",
```

Comment:
maybe we need to add an image here

Reply:
Yep it kind of place holder for now, I dont have yet the detail info about the first season of ETH p2e